packages/validation/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@lionweb/validation",
+  "version": "0.5.0-beta.1",
+  "description": "LionWeb json validation for {Java|Type}Script",
```

Comment:
Twice a "`description`" field. I think I like the second one best.

Reply:
Fixed